queries.sql

```diff
@@ -16,8 +16,115 @@ SELECT * FROM animals WHERE name != 'Gabumon'; --Find all animals not named Gabu
 
 SELECT * FROM animals WHERE weight_kg >= 10.4 AND weight_kg <= 17.3; --Find all animals with a weight between 10.4kg and 17.3kg (including the animals with the weights that equals precisely 10.4kg or 17.3kg)
 
+/* 
+Inside a transaction update the animals table 
+by setting the species column to unspecified. 
+Verify that change was made. Then roll back 
+the change and verify that the species 
+columns went back to 
+the state before the transaction.
+*/
 BEGIN;
 UPDATE animals
 SET species = 'unspecified';
 
```

Comment:
Hello! 👋 Thank you for submitting your code for review. You've done a good job in utilizing SQL transactions, but there's a small gap to fully meet the project requirements. 📝

#### Highlights:
- You've correctly initiated a transaction using `BEGIN`.
- Your `UPDATE` query for setting the `species` column to 'unspecified' is spot on! 👏

#### Areas for Improvement:
- According to the project requirements, you are explicitly asked to include steps to verify that the change was made and then rolled back. Your code lacks the `SELECT` queries that confirm these steps.

Here's how you could improve it:

```sql
BEGIN; -- start transaction

UPDATE animals
SET species = 'unspecified'; -- make change

SELECT species from animals; -- 👈 Add this line to verify that change was made

ROLLBACK; -- roll back the change

SELECT species from animals; -- 👈 Add this line to verify that change was undone
```

Please include these additional `SELECT` queries to fully meet the project requirements. Looking forward to your next commit! 🚀

Best,
Temitope